api/fluentbitoperator/v1alpha2/plugins/filter/Multiline_types.go

```diff
@@ -0,0 +1,38 @@
+package filter
+
+import (
+	"kubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2/plugins"
+	"kubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2/plugins/params"
+)
+
+// +kubebuilder:object:generate:=true
+
+// The Multiline Filter helps to concatenate messages that originally belong to one context but were split across multiple records or log lines.
+type Multiline struct {
+	// The Multi struct helps to concatenate messages that originally belong to one context but were split across multiple records or log lines.
+	Multi *Multi `json:"multiline,omitempty"`
```

Comment:
What is the purpose of `multi`?

Reply:
Because its arguments are `multiline.parser` and `multiline.key_content`. So I created a structure so that I could add new parameters.

Reply:
Maybe `inline` is more suitable for `multi`?

Reply:
Sorry, I am not clear about it. I mean `Mutil` should be `inline` type.

```
type Multiline struct {
  *Multi `json:",inline"`
}
```